### DISCUSSION TITLE
How to create a readable media file

### DISCUSSION THREAD

**Discussion by xiaozhiagi** [zauberzeug/nicegui#1303] [Q&A] [answered]
### Question

My question: I want to create a media folder that can be read by the django framework, storing media files such as images, audio, and videos, which can then be used in ui.markdown.thanks

**Comment by falkoschindler** [comment]
Sorry, @txhy2018, this question is _way_ to broad to be answered here with reasonable effort. I don't even see how this is related to NiceGUI.

**Reply by xiaozhiagi** [reply]
I'm just giving an example, it doesn't require understanding django. My question is: How can I directly display images from folders in ui.markdown, similar to ui.image, which can display local folder images

**Reply by falkoschindler** [reply]
Oh, I see!

**Comment by falkoschindler** [comment]
Currently there's no way to embed images in `ui.markdown` using local file paths. But once you serve the image folder, you can use URL paths like this:

```py
app.add_static_files('/images', 'website/static')
ui.markdown('This is the NiceGUI logo: ![logo](/images/logo.png)')
```

**Reply by xiaozhiagi** [reply]
I created the images folder and website/static folder in the main.py path. The images folder contains logo. ico, and the error message is as follows:

http://127.0.0.1:8080/images/logo.ico not found

**Reply by falkoschindler** [reply, marked as answer]
"/images" isn't supposed to be a folder, but a route which is used to access the images from the browser. In my example I'm using the route "/images/logo.png" to access the file "website/static/logo.png". You can run the script from NiceGUI's root directory, which contains the "website" folder.
